Resources/Prototypes/_CorvaxNext/Entities/Objects/Tools/bodycamera.yml

```
@@ -0,0 +1,25 @@
+- type: entity
+  abstract: true
+  parent: BaseItem
+  id: BaseBodyCamera
+  components:
+  - type: Eye
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)